third_party/lua-yaml/lyaml.cc

```diff
@@ -618,7 +618,7 @@ static int dump_node(struct lua_yaml_dumper *dumper)
    yaml_event_t ev;
    yaml_scalar_style_t style = YAML_PLAIN_SCALAR_STYLE;
    int is_binary = 0;
-   char buf[MAX(FPCONV_G_FMT_BUFSIZE, DT_TO_STRING_BUFSIZE)];
+   char buf[DT_IVAL_TO_STRING_BUFSIZE];
```

Comment:
I would suggest to define a new macro as max of these 3 values. Otherwise you can't guarantee the buffer will always fit all 3 types if the sizes will change in the future.

Reply:
The problem is that these values are defined in different places. I think the best way would be to define them in the same place in the same enum, but not sure if that should be done in this patch-set. So far I have added two static asserts in two places. Also, I increased value of DT_IVAL_TO_STRING_BUFSIZE to 256, since 96 is too small.